hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/client/OzoneQuota.java

```diff
@@ -193,6 +193,11 @@ public static OzoneQuota parseSpaceQuota(String quotaInBytes) {
           break;
         }
       }
+      // there might be no unit specified.
+      if (size.equals("")) {
+        size = uppercase;
+        currUnit = Units.B;
```

Comment:
if size is empty, currUnit will be Units.B already, do we need to re assign it here

Reply:
Done!